src/cudamatrix/cu-device.cc

```diff
@@ -110,6 +110,12 @@ void CuDevice::Initialize() {
     // Initialize CUBLAS.
     CUBLAS_SAFE_CALL(cublasCreate(&cublas_handle_));
     CUBLAS_SAFE_CALL(cublasSetStream(cublas_handle_, cudaStreamPerThread));
+
+    #if CUDA_VERSION >= 9100
```

Comment:
Let's have the `#if` left-justified, I think it's more standard.  Same with other preprocessor directives you come across.

Reply:
pushed